primitives/runtime/src/lib.rs

```diff
@@ -269,18 +269,46 @@ pub enum OwnedBridgeModuleError {
 	Halted,
 }
 
+/// Operating mode for a bridge module.
+pub trait OperatingMode: Send + Copy + Debug + FullCodec {
+	fn is_halted(&self) -> bool;
```

Comment:
Please some docs for this public method

Reply:
No worries! Thanks! :)

Reply:
(btw - I still do not see any docs :) )

Reply:
Yes, I placed it in the implementation by mistake. Fixed.